src/app/page.tsx

```diff
@@ -1,286 +1,17 @@
-              md:text-[32px]
-              lg:text-[50px]'
-            >
-              친구들이 달아준
-              <br />
-              내용을 확인해 봐요
-            </h2>
-          </div>
-          <div className='flex flex-col gap-[10px]'>
-            <LandingChatLight alt='밝은 테마 채팅 화면' />
-            <div
-              className='flex 
-              w-[335px] h-[102px] gap-[10px] 
-              md:w-[648px] md:h-[196px] md:gap-[22px]
-              lg:w-[924px] lg:h-[280px] lg:gap-[40px]'
-            >
-              <LandingNotificationBell
-                alt='알림 종 아이콘'
-                className='bg-secondary-purple-100 border border-none rounded-[10px]
-                w-[102px] aspect-square
-                md:w-[198px]
-                lg:w-[280px]'
-              />
-              <LandingFaqChat
-                alt='FAQ 채팅 버블'
-                className='border border-none rounded-[10px]
-                w-[223px] h-[102px]
-                md:w-[428px] md:h-[198px]
-                lg:w-[604px] lg:h-[280px]'
-              />
-            </div>
-          </div>
-        </section>
-      </main>
-
-      <section className='flex flex-col gap-[40px] bg-grayscale-500 text-center text-white px-[31px] py-[100px]'>
-        <div
-          className=' m-auto
-          w-[335px]
-          md:w-[646px] 
-          lg:w-[924px]'
-        >
-          <h1
-            className='font-nexon-gothic-bold
-          text-[30px]
-          md:text-[60px]'
-          >
-            나만의 위키 만들어 보기
-          </h1>
-          <button
-            className='mx-[40px] my-[40px] px-[20px] py-[15px] rounded-[15px]
-            font-pretendard font-semibold transition cursor-pointer
-            bg-white text-grayscale-500 
-            text-[20px]
-            md:text-[24px]'
-          >
-            지금 시작하기
-          </button>
-        </div>
-      </section>
-
-      <footer
-        className='text-left bg-grayscale-600 text-white font-pretendard
-      px-[20px] py-[40px]'
-      >
-        <div
-          className=' m-auto
-          w-[335px]
-          md:w-[646px] 
-          lg:w-[924px]'
-        >
-          <p
-            className='
-          text-[10px]
-          md:text-[16px]'
-          >
-            Copyright ⓒ Wikied. All Rights Reserved
-          </p>
-          <div
-            className='
-          text-[8px] mt-[10px] mb-[20px]
-          md:text-[14px] md:mb-[30px]'
-          >
-            <p>사업자등록번호 000-00-00000 | 통신판매신고 제2020-서울-00000호 | 대표 : 이지은</p>
-            <p>서울특별시 중구 청계천로 123, 위키드빌딩</p>
-          </div>
-          <div
-            className='flex gap-[30px] 
-          text-[8px] my-[20px]
-          md:text-[14px] 
-          '
-          >
-            <p>서비스 이용 약관</p>
-            <p>개인정보 취급방침</p>
-            <p>전자금융거래 기본약관</p>
-          </div>
-        </div>
-      </footer>
-    </div>
+    <>
+      <Section1 />
+      <Section2 />
+      <Section3 />
+      <Section4 />
```

Comment:
제가 참 좋아하는 분리방식입니다 ㅎ

다만 이름을 이런식으로 바꿔보는건 어떨까요?
```suggestion
      <HeroSection />
      <ShareSection />
      <ViewSection />
      <CTASection />
```